app/admin_statistics.py

```diff
@@ -17,38 +17,30 @@
 import sys
 
 import model
-from utils import *
+import utils
 
-class Handler(BaseHandler):
+
+class Handler(utils.BaseHandler):
+    # Show numbers of new records and notes created in each repository.
     repo_required = False
     admin_required = True
 
     def get(self):
-        """
-        Get all repositories and
-        num_persons and notes for each repository
-        Return: list contains dictionarys, e.g.
-        [{'repo': haiti, 'num_persons': 10, 'num_notes': 5}, {...}]
-        """
         repos = sorted(model.Repo.list())
-        all_usage = []
-        for repo in repos:
-            all_usage.append(self.get_usage(repo))
-        self.render('admin_statistics.html',
-                     all_usage=all_usage)
+        all_usage = [self.get_repo_usage(repo) for repo in repos]
+        self.render('admin_statistics.html', all_usage=all_usage)
 
-    def get_usage(self, repo):
+    def get_repo_usage(self, repo):
         """
         Get num_persons and notes for specific repository
         Return: dictionary contains repository name,
         number of persons and notes, e.g.
         {'repo': haiti, 'num_persons': 10, 'num_notes': 5}
         """
         counters = model.UsageCounter.get(repo)
-        persons = counters.person_counter if counters else 0
-        notes = counters.note_counter if counters else 0
-        usage = {}
-        usage['repo'] = repo
-        usage['num_persons'] = persons
-        usage['num_notes'] = notes
+        usage = {
```

Comment:
Now you can further simplify the code by directly returning this instead of using ```usage``` variable.